### ISSUE TITLE
typo in example `proofScripts.js` in readme

### ISSUE THREAD

**Issue by darcyparker** [ritterim/markdown-proofing#61]
```
else if (arg === 'git-index-and-uncommitted') {
  gitProof([
    `git diff --name-only --diff-filter=ACMRT ${filePattern}`,
    `git diff --name-only --cached' ${filePattern}`
  ]);
}
```
I think it should be:
```
else if (arg === 'git-index-and-uncommitted') {
  gitProof([
    `git diff --name-only --diff-filter=ACMRT ${filePattern}`,
    `git diff --name-only --cached ${filePattern}`
  ]);
}
```
Not sure how the single quote got in there?

**Comment by kendaleiv** [comment]
Thanks for reporting the issue! I fixed it in #62.

**Comment by darcyparker** [comment]
No prob. It's a nice script.
